### PR TITLE
[Refactor] create Python abstract class

### PR DESCRIPTION
**Pull request by wing328**
- create Python abstract class
- minor code format change (Java files)
- only change in the samples (output) is the addition of `title` in the model's properties

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `5.1.x`, `6.0.x`
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

cc @taxpon (2017/07) @frol (2017/07) @mbohlool (2017/07) @cbornet (2017/09) @kenjones-cisco (2017/11) @tomplus (2018/10) @Jyhess (2019/01) @arun-nalla (2019/11) @spacether (2019/11)

### CODE REVIEW

File: docs/generators/python-aiohttp.md [hunk 1]
```diff
@@ -42,23 +42,29 @@ These options may be applied as additional-properties (cli) or configOptions (pl
...
 <li>int</li>
+<li>list</li>
 <li>object</li>
 <li>str</li>
 </ul>
 
 ## RESERVED WORDS
 
 <ul class="column-ul">
+<li>all_params</li>
```
Comment (spacether) [inline docs/generators/python-aiohttp.md:60]: Why is all_params a reserved word?

**Review by spacether — APPROVED** [review]
Looks good; some non-blocking questions.

**Comment by spacether** [comment]
@wing328 does this fix https://github.com/OpenAPITools/openapi-generator/issues/8420?
Could you make this change to fix it?

By eye it looks like the error is happening because we add these files twice
If we stop adding it twice will this be fixed?

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java#L130
openapi-generator/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonLegacyClientCodegen.java

Line 291 in 008ddfb

 supportingFiles.add(new SupportingFile("__init__model.mustache", packagePath() + File.separatorCh

File: docs/generators/python-aiohttp.md (hunk 1 above)
Comment (wing328) [reply docs/generators/python-aiohttp.md:60]: Noticed that too. Will file another PR to clean up the reserved word list.

**Comment by wing328** [comment]
Don't think it will #8420. Will look into it and file a PR if I can come up with a fix.

**Comment by ohhai** [comment]
@wing328  could you please tell what is the current reason to map binary response type to string (not to byte array)?
I mean this comment: https://github.com/OpenAPITools/openapi-generator/issues/4372#issuecomment-862856208

**Pull request merged**